docs/configure.md

```diff
@@ -0,0 +1,153 @@
+# Configuration
+
+## Gather Elasticsearch details
+
+You will need the following details to configure the Elasticsearch integration:
+
+1. The URL of your Elasticsearch instance
+2. Credentials to access the Elasticsearch instance (if required)
+
+### Credentials
+
+You must provide credentials if your Elasticsearch instance is secured. While we support authenticating via username/password, we recommend using API Keys for simplicity and compatibility with all versions of Elasticsearch.
+
+Use the following command to create an API Key for the Home Assistant component.:
+
+=== "curl"
+    ```bash
+    curl https://localhost:9200/_security/api_key \ # (1)
+      -X POST \
+      -H "Content-Type: application/json" \
+      -u elastic:changeme \ # (2)
+      -d'
+      {
+        "name": "home_assistant_component",
+        "role_descriptors": {
+          "hass_writer": {
+            "cluster": [
+              "manage_index_templates",
+              "manage_ilm",
+              "monitor"
+            ],
+            "indices": [
+              {
+                "names": [
+                  "metrics-homeassistant*"
```

Comment:
should be `metrics-homeassistant.*`

Reply:
Done in https://github.com/legrego/homeassistant-elasticsearch/commit/861ac46421ca6043a2f1c73c77669471885039a5